src/components/Theme/style.css

```diff
@@ -108,6 +107,20 @@
   border-radius: 32*@unit;
   padding: 0;
   outline: none;
+
+  /*
+   * Beware! Hack!
```

Comment:
This is the second hack... That really is a hack :/

Reply:
I've refactored this out now, so that we have a `Button` component, which has a text span, so we no longer have this hack!